admin_guide/high_availability.adoc

```diff
@@ -252,16 +261,71 @@ ip addr show $interface | grep -i MULTICAST
 echo "Check multicast groups ... "
 ip maddr show $interface | grep 224.0.0 | grep $interface
 
-echo "Optionally, add accept rule and persist it ... "
-sudo /sbin/iptables -I INPUT -i $interface -d 224.0.0.18/32 -j ACCEPT
-
-echo "Please ensure the above rule is added on system restarts."
-
 EOF
 done;
 ----
 ====
 
+[[options-environment-variables]]
+=== Command Line Options and Environment Variables
+
+.Command Line Options and Environment Variables
+[cols="1a,3a,1a,4a",options="header"]
+|===
+
+| Option | Variable Name | Default | Notes
+
+|--watch-port
+|`*OPENSHIFT_HA_MONITOR_PORT*`
+|80
+|This port must be open for the service to be running.
+
+|--interface
+|`*OPENSHIFT_HA_NETWORK_INTERFACE*`
+|
+|By default all NICs are watched.
+
+|
+|`*OPENSHIFT_HA_REPLICA_COUNT*`
+|2
+|This must match spec.replicas in the DC
+
+|--virtual-ips
+|`*OPENSHIFT_HA_VIRTUAL_IPS*`
+|
+|E.g., 1.2.3.4-6,1.2.3.9
+
+|--vrrp-id-offset
+|`*OPENSHIFT_HA_VRRP_ID_OFFSET*`
+|0
+|When multiple *ipfailover* deployments are used each must have a unique offset.
```

Comment:
Worse than that, it's used as a base, but if you look at config_generators.sh it is incremented once per IP passed... so if your first failover specifies 4 IPs, you need to set the offset to 4 in the next failover config.  SO... I'd recommend incrementing by 10s or more so that people can add IPs to existing configs without messing with the world.  This needs to be explained clearly.